src/main/resources/view/HelpWindow.fxml

```diff
@@ -72,15 +71,15 @@
                         <FXCollections fx:factory="observableArrayList">
                             <UG action="Help" format="help" />
                             <UG action="Exit" format="exit" />
-                            <UG action="Add Owner" format="add owner n/OWNERNAME p/PHONE e/EMAIL a/ADDRESS [t/TAG].../"/>
-                            <UG action="Add Dog" format="add dog n/DOGNAME b/BREED d/DATEOFBIRTH s/SEX o/OWNERID [t/TAG]…"/>
-                            <UG action="Add Program" format="add program n/NAME [s/DATEOFSESSION]... [t/TAG]..."/>
-                            <UG action="Edit Owner" format="edit owner OWNERID n/OWNERNAME p/PHONE e/EMAIL a/ADDRESS [t/TAG]..." />
-                            <UG action="Edit Dog" format="edit dog DOGID n/DOGNAME b/BREED d/DATEOFBIRTH s/SEX o/OWNERID [t/TAG]..." />
-                            <UG action="Edit Program" format="edit program PROGRAMID n/NAME [s/DATEOFSESSION]... [t/TAG]..." />
+                            <UG action="Add Owner" format="add owner n/OWNERNAME p/PHONE e/EMAIL a/ADDRESS [t/TAG]"/>
+                            <UG action="Add Dog" format="add dog n/DOGNAME b/BREED d/DATEOFBIRTH s/SEX o/OWNERID [t/TAG"/>
+                            <UG action="Add Program" format="add program n/NAME [s/DATEOFSESSION] [t/TAG]"/>
+                            <UG action="Edit Owner" format="edit owner OWNERID n/OWNERNAME p/PHONE e/EMAIL a/ADDRESS [t/TAG]" />
+                            <UG action="Edit Dog" format="edit dog DOGID n/DOGNAME b/BREED d/DATEOFBIRTH s/SEX o/OWNERID [t/TAG]" />
+                            <UG action="Edit Program" format="edit program PROGRAMID n/NAME [s/DATEOFSESSION] [t/TAG]" />
```

Comment:
I think this portion is still problematic, the ellipsis is necessary, I think what i meant was text getting cut off due to the column width